test/test_ops.py

```diff
@@ -1577,6 +1577,121 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+    @onlyCUDA
+    def test_opinfo_aten_ops_coverage(self):
+        all_aten_ops = {
+            torch.ops.aten.__getattr__(attr)
+            for attr in dir(torch.ops.aten)
+            if not attr.startswith('__') and attr not in {'name'} and
+            isinstance(torch.ops.aten.__getattr__(attr), torch._ops.OpOverloadPacket)
+        }
+
+        # Copies inputs to inplace operations to avoid inplace modifications
+        #   to leaves requiring gradient
+        def _get_safe_inplace(inplace_variant):
+            @wraps(inplace_variant)
+            def _fn(t, *args, **kwargs):
+                return inplace_variant(t.clone(), *args, **kwargs)
+            return _fn
+
+        seen_aten_ops = set()
+
+        class CollectDispatchMode(torch.utils._python_dispatch.TorchDispatchMode):
+            def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+                seen_aten_ops.add(func.overloadpacket)
+                kwargs = kwargs or {}
+                return func(*args, **kwargs)
+
+        for device in ["cpu", "cuda"]:
```

Comment:
`itertools.product` will let this avoid one level of indentation

Reply:
Why enumerate the device types here instead of instantiating cpu and cuda version of the test?